src/index.html

```diff
@@ -359,9 +359,18 @@ <h3>Some example of queries include:</h3>
                             <p>* To fetch all the variants in a segment region:
                                 <br/> <a href="http://www.ebi.ac.uk/eva/webservices/rest/v1/segments/11:128446-128446/variants?species=hsapiens_grch37" target="_blank">http://www.ebi.ac.uk/eva/webservices/rest/v1/segments/11:128446-128446/variants?species=hsapiens_grch37</a></p>
 
-                            <p><br/> * To fetch all the info of a variant:
+                            <p><br/> * To fetch all the info for a variant by ID:
                                 <br/> <a href="http://www.ebi.ac.uk/eva/webservices/rest/v1/variants/rs666/info?species=hsapiens_grch37" target="_blank">http://www.ebi.ac.uk/eva/webservices/rest/v1/variants/rs666/info?species=hsapiens_grch37</a></p>
 
+                            <p><br/> * To fetch locus info for a variant by <a href="https://www.ncbi.nlm.nih.gov/books/NBK44374/#Content.what_is_the_ncbi_assay_id_or__ss" target="_blank">SS ID</a>:

```

Comment:
I noticed in the preview that some users are not very familiar with the terms RS IDs and SS IDs but they are familiar with RefSnp and SubSnp so we could maybe mention it here too as we do in the swagger of the identifiers